nodejs_conductor/native/Cargo.toml

```diff
@@ -8,7 +8,7 @@ exclude = ["artifacts.json", "index.node"]
 
 [lib]
 name = "holochain_node"
-crate-type = ["dylib"]
+# crate-type = ["dylib"]
```

Comment:
dropping this line got `cargo build` to run (was previously throwing linking error)

let's see if tests pass...

Reply:
We need that though. The node native module has to be compiled into a dynamic c library.

Reply:
why? @sphinxc0re